FifthLesson/VectorProgram/VectorProgram/Controller/VectorHelper.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using VectorProgram.Model;
+
+namespace VectorProgram.Controller
+{
+    public static class VectorHelper
+    {
+        public static double AngleBetweenVectors(Vector ferst, Vector second)
+        {
+            var scalarMultiple = ferst.FirstCoordinate * second.FirstCoordinate + ferst.SecondCoordinate * second.SecondCoordinate + ferst.ThirdCoordinate + second.ThirdCoordinate;
```

Comment:
Rename **scalarMultiple** to **scalarMultiplicationResult** since this variable stores the result of scalar multiplication

Reply:
Corrected

Reply:
Added method **CalculateModuleVector**